targets/arm/include/PeripheralNames.h

```diff
@@ -27,8 +27,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *******************************************************************************
  */
-#ifndef MBED_PERIPHERALNAMES_H
```

Comment:
@jetkov i asked yehia to make chnages to the targets folder. do you think some of these defines might be used by mbed or some othere dependency or is it safe